### PR TITLE
Make the loading animation reveal previous anonymization summary

### PR DESCRIPTION
**Pull request by pdobacz**
Follows up after #262 and discussion therein - it is very useful to see the "previous" anonymization summary, when the new one is calculating after having changed the anonymization parameters. This change accomplishes that
Screenshot from 2021-11-23 11-11-52

The "spinner" look and feel is now the same as for the Schema Load step and the Anonymized Results table.

### CODE REVIEW

File: src/AnonymizationStep/AnonymizationStep.tsx [hunk 1]
```diff
@@ -42,42 +42,45 @@ const emptySummary: AnonymizationSummary = {
...
+            <Text type="secondary">Anonymizing data</Text>
+          </Space>
+        </div>
+      ) : loading ? (
+        <div className="AnonymizationSummary completed notebook-step">
+          <Spin>{summaryDescriptions(summary)}</Spin>
+        </div>
+      ) : (
+        <div className="AnonymizationSummary completed notebook-step">{summaryDescriptions(summary)}</div>
+      )}
```
Comment (edongashi) [inline src/AnonymizationStep/AnonymizationStep.tsx:82]: Can be condenset to `<Spin spinning={loading}>summaryDescriptions(summary)}</Spin>`

File: src/AnonymizationStep/AnonymizationStep.tsx [hunk 2]
```diff
@@ -42,42 +42,45 @@ const emptySummary: AnonymizationSummary = {
...
-        <Descriptions.Item label="Median Distortion">
-          {!loading ? formatPercentage(summary.medianDistortion) : <TextPlaceholder />}
-        </Descriptions.Item>
-        <Descriptions.Item label="Maximum Distortion">
-          {!loading ? formatPercentage(summary.maxDistortion) : <TextPlaceholder />}
-        </Descriptions.Item>
-      </Descriptions>
-    </div>
+      {summary === emptySummary ? (
+        <div className="AnonymizationSummary loading notebook-step">
```
Comment (edongashi) [inline src/AnonymizationStep/AnonymizationStep.tsx:70]: The `notebook-step` here adds margin, and the `<Title>` and nav marker is outside of it. The topmost div should be a div with `AnonymizationSummary notebook-step`

File: src/AnonymizationStep/AnonymizationStep.tsx [hunk 3]
```diff
@@ -63,24 +63,20 @@ function summaryDescriptions(summary: AnonymizationSummary) {
 
 function AnonymizationSummary({ result: { summary }, loading }: CommonProps) {
   return (
-    <>
+    <div className="AnonymizationSummary notebook-step">
       <NotebookNavAnchor step={NotebookNavStep.AnonymizationSummary} status={loading ? 'loading' : 'done'} />
       <Title level={3}>Anonymization summary</Title>
       {summary === emptySummary ? (
-        <div className="AnonymizationSummary loading notebook-step">
+        <div className="AnonymizationSummary loading">
```
Comment (edongashi) [inline src/AnonymizationStep/AnonymizationStep.tsx:70]: To match with our naming style, this should now be something like `AnonymizationSummary-spinner`. Maybe div can be removed and collapsed with the `<Space>` below.

File: src/AnonymizationStep/AnonymizationStep.tsx [hunk 4]
```diff
@@ -42,41 +42,40 @@ const emptySummary: AnonymizationSummary = {
...
-        <Descriptions.Item label="Median Distortion">
-          {!loading ? formatPercentage(summary.medianDistortion) : <TextPlaceholder />}
-        </Descriptions.Item>
-        <Descriptions.Item label="Maximum Distortion">
-          {!loading ? formatPercentage(summary.maxDistortion) : <TextPlaceholder />}
-        </Descriptions.Item>
-      </Descriptions>
+      {summary === emptySummary ? (
+        <div className="AnonymizationSummary loading">
+          <Space direction="vertical">
```
Comment (edongashi) [inline src/AnonymizationStep/AnonymizationStep.tsx:71]: Can you check if `align="center"` works? Then you can drop the wrapper entirely.

File: src/AnonymizationStep/AnonymizationStep.tsx (hunk 3 above)
Comment (pdobacz) [reply src/AnonymizationStep/AnonymizationStep.tsx:70]: I have been basing off of `<div className="SchemaLoadStep notebook-step loading">`, should that be updated as well?

File: src/AnonymizationStep/AnonymizationStep.tsx (hunk 4 above)
Comment (pdobacz) [reply src/AnonymizationStep/AnonymizationStep.tsx:71]: it doesn't :(

**Review by edongashi — APPROVED** [review]

**Pull request merged**